api/DOMParser.json

```diff
@@ -80,7 +80,7 @@
               "version_added": "3.2"
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": true
```

Comment:
```suggestion
              "version_added": "2"
```

Reply:
Does it mean `safari` value should also be fixed?